AK/CMakeLists.txt

```diff
@@ -54,6 +54,9 @@ find_package(simdutf REQUIRED)
 swizzle_target_properties_for_swift(simdutf::simdutf)
 target_link_libraries(AK PRIVATE simdutf::simdutf)
 
+find_package(FastFloat CONFIG REQUIRED)
+target_link_libraries(AK PRIVATE FastFloat::fast_float)
```

Comment:
I don't know what swizzling means, but does this need `swizzle_target_properties_for_swift`?